discovery-provider/plugins/pedalboard/apps/relay/src/scripts/sandbox.ts

```diff
@@ -0,0 +1,59 @@
+  sdk,
+  stagingConfig,
+} from "@audius/sdk";
+import dotenv from "dotenv";
+
+/**
+ * File that you can run via `npm run sandbox` to do some manual testing.
+ */
+export const main = async () => {
+  dotenv.config();
+
+  const inputData =
+    "0xd622c72d0000000000000000000000000000000000000000000000000000000037364e8000000000000000000000000000000000000000000000000000000000000000e00000000000000000000000000000000000000000000000000000000037364e80000000000000000000000000000000000000000000000000000000000000012000000000000000000000000000000000000000000000000000000000000001607ac980efd469f466ff07cbf800844ff41b05b29226a481da8d09fa3194cff88700000000000000000000000000000000000000000000000000000000000004c00000000000000000000000000000000000000000000000000000000000000004557365720000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000065570646174650000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000033c7b22636964223a226261676161696572617174776933666a773632706d78636e663378357179763532743467766a347435367666356e666f66686479337067647061696261222c2264617461223a7b2269735f7665726966696564223a747275652c2269735f6465616374697661746564223a66616c73652c226e616d65223a22746f74616c6c79616c6563222c2268616e646c65223a22746f74616c6c796e6f74616c6563222c2270726f66696c655f70696374757265223a6e756c6c2c2270726f66696c655f706963747572655f73697a6573223a22516d64354a755374544256766471777866334c5a5044356b50794d4c414c425a6d52734b3761695053745a664c72222c22636f7665725f70686f746f223a6e756c6c2c22636f7665725f70686f746f5f73697a6573223a6e756c6c2c2262696f223a22776861742061626f7574206d65222c226c6f636174696f6e223a2244656e7665722c20434f222c226172746973745f7069636b5f747261636b5f6964223a6e756c6c2c2263726561746f725f6e6f64655f656e64706f696e74223a2268747470733a2f2f63726561746f726e6f646531312e73746167696e672e6175646975732e636f2c68747470733a2f2f757365726d657461646174612e73746167696e672e6175646975732e636f2c68747470733a2f2f63726561746f726e6f6465352e73746167696e672e6175646975732e636f222c226173736f6369617465645f77616c6c657473223a6e756c6c2c226173736f6369617465645f736f6c5f77616c6c657473223a6e756c6c2c22636f6c6c65637469626c6573223a6e756c6c2c22706c61796c6973745f6c696272617279223a7b22636f6e74656e7473223a5b7b22706c61796c6973745f6964223a313038333936363637312c2274797065223a22706c61796c697374227d2c7b22706c61796c6973745f6964223a313336303435323137352c2274797065223a22706c61796c697374227d2c7b22706c61796c6973745f6964223a3535352c2274797065223a22706c61796c697374227d5d7d2c226576656e7473223a6e756c6c2c22616c6c6f775f61695f6174747269627574696f6e223a66616c73652c22757365725f6964223a3932363330353932307d7d0000000000000000000000000000000000000000000000000000000000000000000000419d3130f92d40ab9170e59daaf3cb65b52d7dfdde9392d39f8a72f7413b5334c57b9da67e0f88f8cde90d443c4e5f14625b7076c52fd786289e540d9ea004f01c1c00000000000000000000000000000000000000000000000000000000000000";
+  const params = decodeAbi(inputData);
```

Comment:
where is this abi data getting used?

Reply:
ah this is a testing file to ensure it was decoding properly, can remove that